portfolio/src/main/webapp/index.html

```diff
@@ -1,73 +1,188 @@
 <!DOCTYPE html>
```

Comment:
Great job with the website! Code wise, I don't have much to say. You did great! There's a few design things that can be improved, but overall the website's fantastic. I'm going to be a little bit more thorough here, but it's just because you've done a great job here.

1) It might be nice to add buttons that move users to the next section. Then they can be introduced to you, be shown all your work, and then be given the chance to contact you. Right now, you have a button to get in touch before all that. You do have the buttons at the top right, but the most obvious button on the page is "Get in Touch", and it's what I clicked on before anything else. People are like light seeking bugs. They'll just subconsciously go for whatever sticks out the most. Make sure that's the path that is going to lead to the best experience! Alternatively, put both there.
2) The website overall is very sleek and elegant. The pictures however, seem sort of just placed there. I'd recommend using css to crop them or give them borders so they seem more intentionally placed. I'm a fan of the circle border using border-radius, but I'm sure there's plenty of other elegant looking examples you could find on Google.
3) I'm 50/50 on this suggestion, but it might be nice to have a photo of you at the top of the page as well. It's a very elegant and professional style you have for the website, and I think that adding a photo there makes it have a touch of personal as well.